### PR TITLE
WIP: Add page for developer advocates

### PR DESCRIPTION
**Pull request by mattstratton**
This adds a page at `/developer-advocates` which lists the current DAs, along with some other content.

Note: this adds an optional field to the team entries in the `data` directory; anyone with `role = "developer-advocate"` will show up on this page.

We are still working through the content, so this is a draft PR at the moment.

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

### CODE REVIEW

**Review by susanev — COMMENTED** [review]
a few thoughts..
- should this live under /about?
- could we use the contact us form instead of the email link?

**Comment by mattstratton** [comment]
> should this live under /about?
I don't mind that; the only thing I was thinking in my head was an easy to remember URL if we need it to rattle off to people?

> should this be linked to from somewhere on the about page?
I defer to @aaronkao on this based on what his vision is for this page, but I don't mind that either :)

> could we use the contact us form instead of the email link?
I already started to talk to @zchase about how we could do the form instead; I much prefer that myself. I just tossed the email link in so the WIP wasn't so empty, but ideally a form instead of having our email address out there for spam harvesting!

**Comment by susanev** [comment]
cool, @aaronkao when we are adding new pages id love for us to think about the navigation and how they fit into our current navigation structure. happy to help however i can.

**Review by cnunciato — APPROVED** [review]
LGTM, though defer to @aaronkao on the details.

Semi-relatedly, IIRC we've also talked about having a page at https://www.pulumi.com/community (which today still 404s -- 😢) that might be a good place to collect community resources & things; I can imaging something like this living on a page like that, too, or this page being linked _from_ that page, or something -- just figured I'd mention that as something to think about as well.

File: themes/default/content/developer-advocates/_index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Developer Advocates
+meta_desc: Pulumi is a Seattle-based company that enables teams to create, deploy, and manage modern cloud applications and infrastructure.
```
[the remark this answers is not in the document]
Comment (mattstratton) [reply themes/default/content/developer-advocates/_index.md:3]: yah, I didn't update that yet; i'm going to loop back to that once the actual content on the page is finalized

**Comment by mattstratton** [comment]
> LGTM, though defer to @aaronkao on the details.
> 
> Semi-relatedly, IIRC we've also talked about having a page at [pulumi.com/community](https://www.pulumi.com/community) (which today still 404s -- 😢) that might be a good place to collect community resources & things; I can imaging something like this living on a page like that, too, or this page being linked _from_ that page, or something -- just figured I'd mention that as something to think about as well.

Talking to Aaron, we think that having it under `/community` makes a lot of sense. I can update the PR for that.

What do we think about adding an alias in hugo so that `/developer-advocates` works as well as `/community/developer-advocates`? Just for ease of sharing the link.

Unrelated, I really would love if we had a pulumi branded shortlink thing, like `pulu.mi/devadvocates`

(note: `.mi` is not a valid TLD lol)

**Comment by mattstratton** [comment]
OK the latest commit moves the "real" URL of the page to `/community/developer-advocates` but also has `/developer-advocates` redirect to it as well:

http://pulumi-hugo-origin-pr-448-c340738f.s3-website.us-west-2.amazonaws.com/community/developer-advocates

http://pulumi-hugo-origin-pr-448-c340738f.s3-website.us-west-2.amazonaws.com/developer-advocates

We will want to put something inside `/community` at some point too of course, right now [it just makes a relatively empty page](http://pulumi-hugo-origin-pr-448-c340738f.s3-website.us-west-2.amazonaws.com/community/).

**Comment by cnunciato** [comment]
> What do we think about adding an alias in hugo so that /developer-advocates works as well as /community/developer-advocates? Just for ease of sharing the link.

Totally reasonable, alias away. 😄

File: themes/default/layouts/page/developer-advocates.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" "Developer Advocates") }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
+        <p>
+            The Pulumi Developer Advocates do things to help you be awesome. Here's a list of the team
+            and their social media contact info! We would love to help you be super cool and rad!
```
Comment (nimbinatus) [inline themes/default/layouts/page/developer-advocates.html:9]: Suggestion to match the current messaging a bit more, and narrowing the focus a bit so people aren't encouraged to reach out for not-advocate reasons.

```suggestion
            The Pulumi Developer Advocate team wants to help you build, manage, and deploy
            all the things and learn something new along the way. Here's a list of the team
            and their social media. Need help? We would love to help you accomplish your goals!
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Some suggestions on wording

**Comment by mattstratton** [comment]
Ugh! how did I mess this up. I rebased but I guess I should have done a merge commit; those files shouldn't show as changed. Ugh.

**Comment by mattstratton** [comment]
Since my attempts at rebasing just made things a huge mess, I'll try a new PR. Sorry!

**Pull request closed without merging**